src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests4.cs

```diff
@@ -2960,5 +2960,75 @@ public class MatchFailureException : InvalidOperationException
             CompileAndVerify(compilation, expectedOutput: @"3
 MatchFailureException()");
         }
+
+        [Fact]
+        public void RecordOrderOfEvaluation()
```

Comment:
Do you want to add a similar test capturing the order of evaluation of property getters involved in evaluating a property pattern?

Reply:
Those are in this test.